web-app/src/main/webapp/script.js

```diff
@@ -36,8 +36,8 @@ window.onload = async () => {
 
 function initMap() {
   const map = new google.maps.Map(document.getElementById("map"), {
-    center: { lat: -34.397, lng: 150.644 },
-    zoom: 5,
+    center: { lat: 51.5074, lng: -0.1278 },
```

Comment:
Out of curiosity: what are those coordinates? :)

Reply:
London! It's where all our data currently is.